install/gcp/installer/forseti_server_installer.py

```diff
@@ -77,6 +77,14 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
             bool: Whether or not the deployment was successful.
             str: Deployment name.
         """
+        self.has_roles_script = gcloud.grant_server_svc_acct_roles(
```

Comment:
For consistency, can you please make the same change in the `forseti_client_installer.py` as well?  (include the timeout)

Reply:
Added in.